qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_sampling_measurement.py

```diff
@@ -0,0 +1,98 @@
+import warnings
+
+import numpy
+
+from qcodes import ParameterWithSetpoints
```

Comment:
Please don't use the global namespace imports within qcodes.

Reply:
Lazy mistake, taken care of